src/huggingface_hub/inference/_providers/__init__.py

```diff
@@ -166,6 +168,12 @@
     "openai": {
         "conversational": OpenAIConversationalTask(),
     },
+    "ovhcloud": {
+        "conversational": OVHcloudAIEndpointsConversationalTask(),
+        "text-to-image": OVHcloudAIEndpointsTextToImageTask(),
+        "feature-extraction": OVHcloudAIEndpointsFeatureExtractionTask(),
+        "automatic-speech-recognition": OVHcloudAIEndpointsAutomaticSpeechRecognitionTask(),
+    },
```

Comment:
Can we start with only `conversational` and `text-generation` for now to be aligned [with huggingface.js](https://github.com/huggingface/huggingface.js/blob/73bb2da0e27db97420f86b64f668232a732a4418/packages/inference/src/lib/getProviderHelper.ts#L149)?